core/jwt_authentication.py

```diff
@@ -29,6 +31,11 @@ def authenticate(self, request):
         except (jwt.PyJWTError, JSONWebTokenError) as exc:
             raise exceptions.AuthenticationFailed(str(exc)) from exc
 
+        if CoreConfig.is_valid_health_facility_contract_required:
+            if (hasattr(user, 'health_facility') and hasattr(user.health_facility, 'contract_end_date') and
+                    user.health_facility.contract_end_date > datetime.now()):
+                raise exceptions.AuthenticationFailed("HF_CONTRACT_INVALID")
```

Comment:
shouldn't it be 
``` python
raise exceptions.AuthenticationFailed(_("HF_CONTRACT_INVALID"))
```
with translation?

Reply:
No, because I use this as a key for frontend. This is not meant to be displayed to the user.

Reply:
thanks for clarification